addons/binding/org.openhab.binding.nest/ESH-INF/thing/smoke-detector.xml

```diff
@@ -3,30 +3,30 @@
 	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
-	<thing-type id="smoke_detector" listed="false">
+	<thing-type id="smoke_detector">
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="account" />
 		</supported-bridge-type-refs>
 
 		<label>Nest Protect</label>
-		<description>The smoke detector/Nest Protect for the account.</description>
+		<description>The smoke detector/Nest Protect for the account</description>
 
 		<channels>
 			<channel id="ui_color_state" typeId="UiColorState">
 				<label>UI Color</label>
```

Comment:
I just see that there are labels&descriptions on all channels here - this should actually be removed as label&desc should be defined on the type already.

Reply:
This was actually still on my todo list. But now they're gone!